.github/workflows/ci.yml

```diff
@@ -602,6 +608,8 @@ jobs:
     needs:
       - parachain-build-dev
       - tee-build
+      - set-condition
+    if: needs.set-condition.outputs.run_update_client_api
```

Comment:
Will this work here? I feel like it won't work. Should we` if: needs.set-condition.outputs.run_update_client_api== 'true'` :p

Reply:
yeah, good catch missed that one